content/reference/storage/namespaces.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Namespaces'
+weight: 80
+---
+
+While Workers KV is an account-level feature, you may not want every
+key-value pair to be globally available. You can create up to a hundred
+*namespaces* in your account, and then bind that namespace to your script. The
+*script will then only have access to that specific namespace, and will read
```

Comment:
Looks like there might be a syntax error here? I think your editor thought you were making a list 😅

Reply:
oof, thanks. i looked at the pages visually after rendering to try and catch these things... oh well